ci/cloudbuild/builds/publish-docs.sh

```diff
@@ -50,11 +56,13 @@ if [[ "${TRIGGER_TYPE}" == "ci" ]]; then
   bucket="docs-staging"
 fi
 
-export CC=clang
-export CXX=clang++
 cmake -GNinja "${doc_args[@]}" -S . -B cmake-out
+cmake --build cmake-out --target install
 cmake --build cmake-out --target doxygen-docs
 
+# Cleans up the installed artifacts, which are no longer needed.
+test -d "${install_dir}" && rm -rf "${install_dir}"
```

Comment:
The `test -d` looks unnecessary.

Reply:
It was. It was an attempt to make the `rm -rf` a bit safer, like in case `${install_dir}` was empty. But in that case, there's nothing to guard against. So, removed.